src/styles/variables/_ndb-light-theme.scss

```diff
@@ -3,18 +3,77 @@
  */
 @use "@angular/material" as mat;
 
+$primary-palette: (
+        50: var(--primary-50, map-get(mat.$orange-palette, 50)),
+        100: var(--primary-100, map-get(mat.$orange-palette, 100)),
+        200: var(--primary-200, map-get(mat.$orange-palette, 200)),
+        300: var(--primary-300, map-get(mat.$orange-palette, 300)),
+        400: var(--primary-400, map-get(mat.$orange-palette, 400)),
+        //500: map-get(mat.$orange-palette, 500),
+        // TODO somehow this CSS variable breaks the progress bars
+        500: var(--primary-500, map-get(mat.$orange-palette, 500)),
+        600: var(--primary-600, map-get(mat.$orange-palette, 600)),
+        700: var(--primary-700, map-get(mat.$orange-palette, 700)),
+        800: var(--primary-800, map-get(mat.$orange-palette, 800)),
+        900: var(--primary-900, map-get(mat.$orange-palette, 900)),
+        A100: var(--primary-A100, map-get(mat.$orange-palette, A100)),
+        A200: var(--primary-A200, map-get(mat.$orange-palette, A200)),
+        A400: var(--primary-A400, map-get(mat.$orange-palette, A400)),
+        A700: var(--primary-A700, map-get(mat.$orange-palette, A700)),
+        contrast: map-get(mat.$orange-palette, contrast)
```

Comment:
Contrast colors are not configurable and not adjusted based on the selected theme.